src/egithub_webhook.erl

```diff
@@ -73,30 +75,58 @@ write_comments(Cred, Repo, PR, Comments, Messages) ->
           position  := Position,
           text      := Text
          }) ->
-      write_comment(Cred, Repo, PR, CommitId, Path, Position, Text, Comments)
+            write_line_comment(Cred, Repo, PR, CommitId,
+                               Path, Position, Text, Comments);
+       (#{text := Text,
+          position := 0
```

Comment:
You are right, although this works because the first clause won't match a map that doesn't have keys `commit_id` and `path`, and all maps that don't have those keys have `position := 0`, but that's just for now, so I will fix that.